concepticondata/conceptlists.tsv

```diff
@@ -262,5 +262,6 @@ Luniewska-2019-299	Łuniewska, Magdalena AND Wodniecka, Zofia AND Miller, Carol
 Pereira-2018-180	Pereira, Francisco and Lou, Bin and Pritchett, Brianna and Ritter, Samuel and Gershman, Samuel J. and Kanwisher, Nancy and Botvinick, Matthew and Fedorenko, Evelina	2018		180	specific	English	English	https://www.nature.com/articles/s41467-018-03068-4	Pereira2018		The given list was compiled in the frameworks of research on the neural basis of semantic representations. As reported by the authors, it resulted from clustering words from [massive text corpora](:bib:Brysbaert2014) into 180 semantic groups, covering the entire semantic space of the corpora. The most representative concepts, manually selected from each cluster, were then used as input stimuli in three fMRI experiments in order to investigate the relationship between the linguistic representations and the resulting imaging data. The researchers report that semantic information is not confined to a specific brain region, as commonly assumed, but rather distributed across several high-level cortical networks.	1-13	
 Dunn-2017-12	Dunn, Michael and Dewey, Tonya Kim and Arnett, Carlee and Eythórsson, Thórhallur and Barðdal, Jóhanna	2017		12	historical	English	Germanic	https://muse.jhu.edu/article/651291	Dunn2017		This list was compiled in the development of the model, accounting for the evolutionary change in noncanonically case-marked argument structure constructions in the Germanic languages. The data for this study come from the database compiled by the NonCanCase project at the University of Bergen and the EVALISA project at Ghent University and represent twelve cognate sets of predicates with dative- and accusative-subject marking from Proto-Germanic to the modern languages. Additionally, the dataset contains the information on the chronological order of the cases, assigned to the subject by each verb.	e1-e22	
 Stoll-1884-259	Stoll, Otto	1884		259	basic	German	Mayan languages		Stoll1884		This list underlies the comparative study of the 16 Mayan languages, spoken outside the Republic of Guatemala. Its vocabulary was originally compiled and the corresponding data were collected by Berendt around the middle of the 19th century, later published by Stoll in 1884 with minor modifications. The languages are ordered according to the degree of their relatedness to the Mayan family, starting from the most genetically remote variety. It is also worth noting that the variety of Chol, recorded by Berendt, is reported to have undergone some significant changes after the point of its documentation.		
+Chan-2019-95	Chan, Eugene S. L.	2019		95	questionnaire	English	Global	https://mpi-lingweb.shh.mpg.de/numeral/	Chan2019		This list is derived from the extensive database, underlying the long-running project ["Numeral Systems of the World's Languages"](https://mpi-lingweb.shh.mpg.de/numeral/), and represents the numeral types, encountered across roughly 7,000 languages.
```

Comment:
Change wording here "This list is derived from the extensive database project link-to-numerals and represents the numeral types across roughly ..."